README
NAME

    Dist::Zilla::PluginBundle::Author::LESPEA - LESPEA's Dist::Zilla
    Configuration

VERSION

    version 1.007000

SYNOPSIS

        #  In dist.ini:
        [@Author::LESPEA]

DESCRIPTION

    This plugin bundle, in its default configuration, is equivalent to:

        [ArchiveRelease]
        [Authority]
        [AutoMetaResources]
        [AutoPrereqs]
        [ConfirmRelease]
        [ConsistentVersionTest]
        [CopyFilesFromBuild]
        [DualBuilders]
        [ExecDir]
        [ExtraTests]
        [FakeRelease]
        [GatherDir]
        [HasVersionTests]
        [InstallGuide]
        [License]
        [MakeMaker]
        [ManifestSkip]
        [Manifest]
        [MetaConfig]
        [MetaJSON]
        [MetaNoIndex]
        [MetaTests]
        [MetaYAML]
        [MinimumPerl]
        [ModuleBuild]
        [NextRelease]
        [NoTabsTests]
        [PkgVersion]
        [PodCoverageTests]
        [PodSyntaxTests]
        [PodWeaver]
        [PortabilityTests]
        [PruneCruft]
        [ReportVersions::Tiny]
        [ShareDir]
        [SynopsisTests]
        [Test::CPAN::Changes]
        [Test::Compile]
        [Test::DistManifest]
        [Test::MinimumVersion]
        [Test::Perl::Critic]
        [TestRelease]
        [UploadToCPAN]

OPTIONS

 -remove

    This option can be used to remove specific plugins from the bundle. It
    can be used multiple times.

    Obviously, the default is not to remove any plugins.

    Example:

        ; Remove these two plugins from the bundle
        -remove = CriticTests
        -remove = SynopsisTests

 copy_file, move_file

    If you want to copy or move files out of the build dir and into the
    distribution dir, use these two options to specify those files. Both of
    these options can be specified multiple times.

    The most common reason to use this would be to put automatically
    generated files under version control. For example, Github likes to see
    a README file in your distribution, but if your README file is
    auto-generated during the build, you need to copy each newly-generated
    README file out of its build directory in order for Github to see it.

    If you want to include an auto-generated file in your distribution but
    you don't want to include it in the build, use move_file instead of
    copy_file.

    Example:

        copy_file = README
        move_file = README.pod
        copy_file = README.txt

 release

    This option chooses the type of release to do. The default is 'real,'
    which means "really upload the release to CPAN" (i.e. load the
    UploadToCPAN plugin). You can set it to 'fake,' in which case the
    FakeRelease plugin will be loaded, which simulates the release process
    without actually doing anything. You can also set it to 'none' if you
    do not want this module to load any release plugin, in which case your
    dist.ini file should load a release plugin directly. Any other value
    for this option will be interpreted as a release plugin name to be
    loaded.

    Examples:

        ; Release to CPAN for real (default)
        release = real
        ; For testing, you can do fake releases
        release = fake
        ; Or you can choose no release plugin
        release = none
        ; Or you can specify a specific release plugin.
        release = OtherReleasePlugin

 archive, archive_directory

    If set to true, the archive option copies each released version of the
    module to an archive directory, using the ArchiveRelease plugin. This
    is the default. The name of the archive directory is specified using
    archive_directory, which is releases by default.

    Examples:

        ; archive each release to the "releases" directory
        archive = true
        archive_directory = releases
        ; Or don't archive
        archive = false

 include_dotfiles

    If this is set to true (not the default), then any file that includes a
    leading '.' will be included in the package

    Example:

        include_dotfiles = true

 compile_synopsis

    If this is set to true (the default), then the SynopsisTests plugin
    will be enabled. This plugin checks the perl syntax of the SYNOPSIS
    sections of your modules. Obviously, if your SYNOPSIS section is not
    perl code (case in point: this module), you should set this to false.

    Example:

        compile_synopsis = false

 add_meta

    If this is set to true (the default), then the AutoMetaResources and
    Authority plugins will be enabled. These plugins adds various metatdata
    such as the github repo, cpan links, etc to the metadata of the plugin.

    Example:

        add_meta = false

SEE ALSO

    Please see those modules/websites for more information related to this
    module.

      * Dist::Zilla::Plugin::ArchiveRelease

      * Dist::Zilla::Plugin::Authority

      * Dist::Zilla::Plugin::AutoMetaResources

      * Dist::Zilla::Plugin::AutoPrereqs

      * Dist::Zilla::Plugin::ConfirmRelease

      * Dist::Zilla::Plugin::ConsistentVersionTest

      * Dist::Zilla::Plugin::CopyFilesFromBuild

      * Dist::Zilla::Plugin::DualBuilders

      * Dist::Zilla::Plugin::ExecDir

      * Dist::Zilla::Plugin::ExtraTests

      * Dist::Zilla::Plugin::FakeRelease

      * Dist::Zilla::Plugin::GatherDir

      * Dist::Zilla::Plugin::HasVersionTests

      * Dist::Zilla::Plugin::InstallGuide

      * Dist::Zilla::Plugin::License

      * Dist::Zilla::Plugin::MakeMaker

      * Dist::Zilla::Plugin::Manifest

      * Dist::Zilla::Plugin::ManifestSkip

      * Dist::Zilla::Plugin::MetaConfig

      * Dist::Zilla::Plugin::MetaJSON

      * Dist::Zilla::Plugin::MetaNoIndex

      * Dist::Zilla::Plugin::MetaTests

      * Dist::Zilla::Plugin::MetaYAML

      * Dist::Zilla::Plugin::MinimumPerl

      * Dist::Zilla::Plugin::ModuleBuild

      * Dist::Zilla::Plugin::NextRelease

      * Dist::Zilla::Plugin::PkgVersion

      * Dist::Zilla::Plugin::PodCoverageTests

      * Dist::Zilla::Plugin::PodSyntaxTests

      * Dist::Zilla::Plugin::PodWeaver

      * Dist::Zilla::Plugin::PruneCruft

      * Dist::Zilla::Plugin::ReadmeAnyFromPod

      * Dist::Zilla::Plugin::ReportVersions::Tiny

      * Dist::Zilla::Plugin::ShareDir

      * Dist::Zilla::Plugin::SpellingCommonMistakesTests

      * Dist::Zilla::Plugin::Test::CPAN::Changes

      * Dist::Zilla::Plugin::Test::Compile

      * Dist::Zilla::Plugin::Test::DistManifest

      * Dist::Zilla::Plugin::Test::MinimumVersion

      * Dist::Zilla::Plugin::Test::Perl::Critic

      * Dist::Zilla::Plugin::Test::Portability

      * Dist::Zilla::Plugin::Test::Synopsis

      * Dist::Zilla::Plugin::TestRelease

      * Dist::Zilla::Plugin::UploadToCPAN

INSTALLATION

    See perlmodinstall for information and options on installing Perl
    modules.

AUTHOR

    Adam Lesperance <lespea@gmail.com>

SUPPORT

 Perldoc

    You can find documentation for this module with the perldoc command.

      perldoc Dist::Zilla::PluginBundle::Author::LESPEA

 Websites

    The following websites have more information about this module, and may
    be of help to you. As always, in addition to those websites please use
    your favorite search engine to discover more resources.

      * MetaCPAN

      A modern, open-source CPAN search engine, useful to view POD in HTML
      format.

      http://metacpan.org/release/Dist-Zilla-PluginBundle-Author-LESPEA

      * Search CPAN

      The default CPAN search engine, useful to view POD in HTML format.

      http://search.cpan.org/dist/Dist-Zilla-PluginBundle-Author-LESPEA

      * RT: CPAN's Bug Tracker

      The RT ( Request Tracker ) website is the default bug/issue tracking
      system for CPAN.

      http://rt.cpan.org/NoAuth/Bugs.html?Dist=Dist-Zilla-PluginBundle-Auth
      or-LESPEA

      * AnnoCPAN

      The AnnoCPAN is a website that allows community annotations of Perl
      module documentation.

      http://annocpan.org/dist/Dist-Zilla-PluginBundle-Author-LESPEA

      * CPAN Ratings

      The CPAN Ratings is a website that allows community ratings and
      reviews of Perl modules.

      http://cpanratings.perl.org/d/Dist-Zilla-PluginBundle-Author-LESPEA

      * CPAN Forum

      The CPAN Forum is a web forum for discussing Perl modules.

      http://cpanforum.com/dist/Dist-Zilla-PluginBundle-Author-LESPEA

      * CPANTS

      The CPANTS is a website that analyzes the Kwalitee ( code metrics )
      of a distribution.

      http://cpants.perl.org/dist/overview/Dist-Zilla-PluginBundle-Author-L
      ESPEA

      * CPAN Testers

      The CPAN Testers is a network of smokers who run automated tests on
      uploaded CPAN distributions.

      http://www.cpantesters.org/distro/D/Dist-Zilla-PluginBundle-Author-LE
      SPEA

      * CPAN Testers Matrix

      The CPAN Testers Matrix is a website that provides a visual overview
      of the test results for a distribution on various Perls/platforms.

      http://matrix.cpantesters.org/?dist=Dist-Zilla-PluginBundle-Author-LE
      SPEA

      * CPAN Testers Dependencies

      The CPAN Testers Dependencies is a website that shows a chart of the
      test results of all dependencies for a distribution.

      http://deps.cpantesters.org/?module=Dist::Zilla::PluginBundle::Author
      ::LESPEA

 Bugs / Feature Requests

    Please report any bugs or feature requests by email to
    bug-dist-zilla-pluginbundle-author-lespea at rt.cpan.org, or through
    the web interface at
    http://rt.cpan.org/NoAuth/ReportBug.html?Queue=Dist-Zilla-PluginBundle-
    Author-LESPEA. You will be automatically notified of any progress on
    the request by the system.

 Source Code

    The code is open to the world, and available for you to hack on. Please
    feel free to browse it and play with it, or whatever. If you want to
    contribute patches, please send me a diff or prod me to pull from your
    repository :)

    https://github.com/lespea/dist-zilla-pluginbundle-author-lespea

      git clone git://github.com/lespea/dist-zilla-pluginbundle-author-lespea.git

COPYRIGHT AND LICENSE

    This software is copyright (c) 2014 by Adam Lesperance.

    This is free software; you can redistribute it and/or modify it under
    the same terms as the Perl 5 programming language system itself.

DISCLAIMER OF WARRANTY

    BECAUSE THIS SOFTWARE IS LICENSED FREE OF CHARGE, THERE IS NO WARRANTY
    FOR THE SOFTWARE, TO THE EXTENT PERMITTED BY APPLICABLE LAW. EXCEPT
    WHEN OTHERWISE STATED IN WRITING THE COPYRIGHT HOLDERS AND/OR OTHER
    PARTIES PROVIDE THE SOFTWARE "AS IS" WITHOUT WARRANTY OF ANY KIND,
    EITHER EXPRESSED OR IMPLIED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
    WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE. THE
    ENTIRE RISK AS TO THE QUALITY AND PERFORMANCE OF THE SOFTWARE IS WITH
    YOU. SHOULD THE SOFTWARE PROVE DEFECTIVE, YOU ASSUME THE COST OF ALL
    NECESSARY SERVICING, REPAIR, OR CORRECTION.

    IN NO EVENT UNLESS REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING
    WILL ANY COPYRIGHT HOLDER, OR ANY OTHER PARTY WHO MAY MODIFY AND/OR
    REDISTRIBUTE THE SOFTWARE AS PERMITTED BY THE ABOVE LICENCE, BE LIABLE
    TO YOU FOR DAMAGES, INCLUDING ANY GENERAL, SPECIAL, INCIDENTAL, OR
    CONSEQUENTIAL DAMAGES ARISING OUT OF THE USE OR INABILITY TO USE THE
    SOFTWARE (INCLUDING BUT NOT LIMITED TO LOSS OF DATA OR DATA BEING
    RENDERED INACCURATE OR LOSSES SUSTAINED BY YOU OR THIRD PARTIES OR A
    FAILURE OF THE SOFTWARE TO OPERATE WITH ANY OTHER SOFTWARE), EVEN IF
    SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH
    DAMAGES.



Changes
Revision history for Dist-Zilla-PluginBundle-Author-LESPEA

1.007000  2014-03-06 14:43:21-06:00 America/Chicago
    [Enhancement]
    *  Improve the skeleton gitignore file

    [Bugfix]
    *   We can no longer have a "default" key that is an empty array as Config::MVP croaks on it

1.006000  2012-09-07 07:38:46 America/Chicago
    [Enhancement]
    *   Pod weaver bundle now collects constants via the 'const' keyword

1.005001  2012-09-05 01:05:27 America/Chicago
    [Bugfix]
    *   EOL tests are horribly broken in windows

1.005000  2012-08-31 15:55:48 America/Chicago
    [Enhancement]
    *   Remove Kwalitee & unusedvars since they're very buggy
    *   Removed 'Miscellanea::RequireRcsKeywords' from shared perlcritic.rc since it's removed in:
    *   Require version 1.118 of Perl::Critic

1.004000  2011-09-22 10:47:24 America/Chicago
    [Bugfix]
    *   Change varios test names as they have been deprecated
    *   Updated test dependencies to the versions that have the new names

1.003000  2011-06-21 00:26:57 America/Chicago
    [Enhancement]
    *   Upgrade to the newest version of Perl::Critic::Bangs and remove deprecated modules
    *   Switch from CPANChangesTests to Test::CPAN::Changes
    *   Remove unnecessary dependencies -- doesn't build on ActiveState anyway since they have crazy short timeouts
    *   Fix Perl::Critic errors

1.002001  2011-03-23 11:34:41 America/Chicago
    [Bugfix]
    *   More manual dependency additions/rearranging to try and get ActiveState to work

    [Enhancement]
    *   Tell cpan not to index README.html


1.002000  2011-03-17 06:15:00 America/Chicago
    [Enhancement]
    *   Add SpellingCommonMistakesTests


1.001000  2011-03-17 05:11:54 America/Chicago
    [Bugfix]
    *   Add more manual dependencies to try fix the cluster that is ActiveState

    [Enhancement]
    *   Remove Perl::Tidy plugin (keep tidy config file in template though)
    *   Switch to ReportVersions::Tiny
    *   Remove CheckChanges since it breaks ReportVersions::Tiny

    [Minor]
    *   All option examples should be code formatted to preserve formatting


1.000000  2011-02-19 17:35:20 America/Chicago
    *   Initial version (heavily modified from RTHOMPSON & KENTNL's bundle)


Build.PL

# This file was automatically generated by Dist::Zilla::Plugin::ModuleBuild v5.013.
use strict;
use warnings;

use Module::Build 0.3601;


my %module_build_args = (
  "build_requires" => {
    "Dist::Zilla::Plugin::Bootstrap::lib" => 0,
    "Module::Build" => "0.3601"
  },
  "configure_requires" => {
    "File::ShareDir::Install" => "0.03",
    "Module::Build" => "0.3601"
  },
  "dist_abstract" => "LESPEA's Dist::Zilla Configuration",
  "dist_author" => [
    "Adam Lesperance <lespea\@gmail.com>"
  ],
  "dist_name" => "Dist-Zilla-PluginBundle-Author-LESPEA",
  "dist_version" => "1.007000",
  "license" => "perl",
  "module_name" => "Dist::Zilla::PluginBundle::Author::LESPEA",
  "recommends" => {},
  "recursive_test_files" => 1,
  "requires" => {
    "CPAN::Meta::YAML" => 0,
    "Carp" => 0,
    "Dist::Zilla" => "4.300000",
    "Dist::Zilla::Plugin::ArchiveRelease" => 0,
    "Dist::Zilla::Plugin::Authority" => 0,
    "Dist::Zilla::Plugin::AutoMetaResources" => 0,
    "Dist::Zilla::Plugin::AutoPrereqs" => 0,
    "Dist::Zilla::Plugin::ConfirmRelease" => 0,
    "Dist::Zilla::Plugin::ConsistentVersionTest" => 0,
    "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
    "Dist::Zilla::Plugin::DualBuilders" => 0,
    "Dist::Zilla::Plugin::EOLTests" => 0,
    "Dist::Zilla::Plugin::ExecDir" => 0,
    "Dist::Zilla::Plugin::ExtraTests" => 0,
    "Dist::Zilla::Plugin::FakeRelease" => 0,
    "Dist::Zilla::Plugin::GatherDir" => 0,
    "Dist::Zilla::Plugin::HasVersionTests" => 0,
    "Dist::Zilla::Plugin::InstallGuide" => 0,
    "Dist::Zilla::Plugin::License" => 0,
    "Dist::Zilla::Plugin::MakeMaker" => 0,
    "Dist::Zilla::Plugin::Manifest" => 0,
    "Dist::Zilla::Plugin::ManifestSkip" => 0,
    "Dist::Zilla::Plugin::MetaConfig" => 0,
    "Dist::Zilla::Plugin::MetaJSON" => 0,
    "Dist::Zilla::Plugin::MetaNoIndex" => 0,
    "Dist::Zilla::Plugin::MetaTests" => 0,
    "Dist::Zilla::Plugin::MetaYAML" => 0,
    "Dist::Zilla::Plugin::MinimumPerl" => 0,
    "Dist::Zilla::Plugin::ModuleBuild" => 0,
    "Dist::Zilla::Plugin::NextRelease" => 0,
    "Dist::Zilla::Plugin::NoTabsTests" => 0,
    "Dist::Zilla::Plugin::PkgVersion" => 0,
    "Dist::Zilla::Plugin::PodCoverageTests" => 0,
    "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
    "Dist::Zilla::Plugin::PodWeaver" => 0,
    "Dist::Zilla::Plugin::PruneCruft" => 0,
    "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
    "Dist::Zilla::Plugin::ReportVersions::Tiny" => 0,
    "Dist::Zilla::Plugin::ShareDir" => 0,
    "Dist::Zilla::Plugin::SpellingCommonMistakesTests" => 0,
    "Dist::Zilla::Plugin::Test::CPAN::Changes" => "0.005",
    "Dist::Zilla::Plugin::Test::Compile" => "1.112400",
    "Dist::Zilla::Plugin::Test::DistManifest" => "v2.0.1",
    "Dist::Zilla::Plugin::Test::MinimumVersion" => "v2.0.1",
    "Dist::Zilla::Plugin::Test::Perl::Critic" => "2.112410",
    "Dist::Zilla::Plugin::Test::Portability" => "v2.0.1",
    "Dist::Zilla::Plugin::Test::Synopsis" => "v2.0.1",
    "Dist::Zilla::Plugin::TestRelease" => 0,
    "Dist::Zilla::Plugin::UploadToCPAN" => 0,
    "Dist::Zilla::Role::MintingProfile::ShareDir" => 0,
    "Dist::Zilla::Role::PluginBundle::Easy" => 0,
    "Moose" => "2.0",
    "Perl::Critic" => "1.118",
    "Perl::Critic::Bangs" => "1.08",
    "Perl::Critic::Lax" => 0,
    "Perl::Critic::Pulp" => 0,
    "Perl::Critic::StricterSubs" => 0,
    "Perl::Critic::Swift" => 0,
    "Perl::MinimumVersion" => 0,
    "Perl::OSType" => 0,
    "Perl::PrereqScanner" => 0,
    "Pod::Coverage::TrustPod" => 0,
    "Pod::Elemental::Transformer::List" => 0,
    "Pod::Spell" => 0,
    "Pod::Weaver" => "3.101632",
    "Pod::Weaver::Config::Assembler" => 0,
    "Pod::Weaver::Section::Installation" => 0,
    "Pod::Weaver::Section::SeeAlso" => 0,
    "Pod::Weaver::Section::Support" => 0,
    "Pod::Weaver::Section::WarrantyDisclaimer" => 0,
    "Test::CPAN::Meta" => 0,
    "Test::Perl::Critic" => 0,
    "Test::Pod::Content" => 0,
    "namespace::autoclean" => 0,
    "perl" => "5.010",
    "strict" => 0,
    "utf8" => 0,
    "warnings" => 0
  },
  "script_files" => [],
  "share_dir" => {
    "dist" => "share",
    "module" => {
      "Dist::Zilla::MintingProfile::Author::LESPEA" => "share/profiles"
    }
  },
  "test_requires" => {
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.88",
    "perl" => "5.010"
  }
);


my %fallback_build_requires = (
  "Dist::Zilla::Plugin::Bootstrap::lib" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Module::Build" => "0.3601",
  "Test::More" => "0.88",
  "perl" => "5.010"
);


unless ( eval { Module::Build->VERSION(0.4004) } ) {
  delete $module_build_args{test_requires};
  $module_build_args{build_requires} = \%fallback_build_requires;
}

my $build = Module::Build->new(%module_build_args);

$build->create_build_script;


Makefile.PL

# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.013.
use strict;
use warnings;

use 5.010;

use ExtUtils::MakeMaker 6.30;

use File::ShareDir::Install;
install_share dist => "share";
install_share module => "Dist::Zilla::MintingProfile::Author::LESPEA", "share\/profiles";


my %WriteMakefileArgs = (
  "ABSTRACT" => "LESPEA's Dist::Zilla Configuration",
  "AUTHOR" => "Adam Lesperance <lespea\@gmail.com>",
  "BUILD_REQUIRES" => {
    "Dist::Zilla::Plugin::Bootstrap::lib" => 0,
    "Module::Build" => "0.3601"
  },
  "CONFIGURE_REQUIRES" => {
    "File::ShareDir::Install" => "0.03",
    "Module::Build" => "0.3601"
  },
  "DISTNAME" => "Dist-Zilla-PluginBundle-Author-LESPEA",
  "EXE_FILES" => [],
  "LICENSE" => "perl",
  "NAME" => "Dist::Zilla::PluginBundle::Author::LESPEA",
  "PREREQ_PM" => {
    "CPAN::Meta::YAML" => 0,
    "Carp" => 0,
    "Dist::Zilla" => "4.300000",
    "Dist::Zilla::Plugin::ArchiveRelease" => 0,
    "Dist::Zilla::Plugin::Authority" => 0,
    "Dist::Zilla::Plugin::AutoMetaResources" => 0,
    "Dist::Zilla::Plugin::AutoPrereqs" => 0,
    "Dist::Zilla::Plugin::ConfirmRelease" => 0,
    "Dist::Zilla::Plugin::ConsistentVersionTest" => 0,
    "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
    "Dist::Zilla::Plugin::DualBuilders" => 0,
    "Dist::Zilla::Plugin::EOLTests" => 0,
    "Dist::Zilla::Plugin::ExecDir" => 0,
    "Dist::Zilla::Plugin::ExtraTests" => 0,
    "Dist::Zilla::Plugin::FakeRelease" => 0,
    "Dist::Zilla::Plugin::GatherDir" => 0,
    "Dist::Zilla::Plugin::HasVersionTests" => 0,
    "Dist::Zilla::Plugin::InstallGuide" => 0,
    "Dist::Zilla::Plugin::License" => 0,
    "Dist::Zilla::Plugin::MakeMaker" => 0,
    "Dist::Zilla::Plugin::Manifest" => 0,
    "Dist::Zilla::Plugin::ManifestSkip" => 0,
    "Dist::Zilla::Plugin::MetaConfig" => 0,
    "Dist::Zilla::Plugin::MetaJSON" => 0,
    "Dist::Zilla::Plugin::MetaNoIndex" => 0,
    "Dist::Zilla::Plugin::MetaTests" => 0,
    "Dist::Zilla::Plugin::MetaYAML" => 0,
    "Dist::Zilla::Plugin::MinimumPerl" => 0,
    "Dist::Zilla::Plugin::ModuleBuild" => 0,
    "Dist::Zilla::Plugin::NextRelease" => 0,
    "Dist::Zilla::Plugin::NoTabsTests" => 0,
    "Dist::Zilla::Plugin::PkgVersion" => 0,
    "Dist::Zilla::Plugin::PodCoverageTests" => 0,
    "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
    "Dist::Zilla::Plugin::PodWeaver" => 0,
    "Dist::Zilla::Plugin::PruneCruft" => 0,
    "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
    "Dist::Zilla::Plugin::ReportVersions::Tiny" => 0,
    "Dist::Zilla::Plugin::ShareDir" => 0,
    "Dist::Zilla::Plugin::SpellingCommonMistakesTests" => 0,
    "Dist::Zilla::Plugin::Test::CPAN::Changes" => "0.005",
    "Dist::Zilla::Plugin::Test::Compile" => "1.112400",
    "Dist::Zilla::Plugin::Test::DistManifest" => "v2.0.1",
    "Dist::Zilla::Plugin::Test::MinimumVersion" => "v2.0.1",
    "Dist::Zilla::Plugin::Test::Perl::Critic" => "2.112410",
    "Dist::Zilla::Plugin::Test::Portability" => "v2.0.1",
    "Dist::Zilla::Plugin::Test::Synopsis" => "v2.0.1",
    "Dist::Zilla::Plugin::TestRelease" => 0,
    "Dist::Zilla::Plugin::UploadToCPAN" => 0,
    "Dist::Zilla::Role::MintingProfile::ShareDir" => 0,
    "Dist::Zilla::Role::PluginBundle::Easy" => 0,
    "Moose" => "2.0",
    "Perl::Critic" => "1.118",
    "Perl::Critic::Bangs" => "1.08",
    "Perl::Critic::Lax" => 0,
    "Perl::Critic::Pulp" => 0,
    "Perl::Critic::StricterSubs" => 0,
    "Perl::Critic::Swift" => 0,
    "Perl::MinimumVersion" => 0,
    "Perl::OSType" => 0,
    "Perl::PrereqScanner" => 0,
    "Pod::Coverage::TrustPod" => 0,
    "Pod::Elemental::Transformer::List" => 0,
    "Pod::Spell" => 0,
    "Pod::Weaver" => "3.101632",
    "Pod::Weaver::Config::Assembler" => 0,
    "Pod::Weaver::Section::Installation" => 0,
    "Pod::Weaver::Section::SeeAlso" => 0,
    "Pod::Weaver::Section::Support" => 0,
    "Pod::Weaver::Section::WarrantyDisclaimer" => 0,
    "Test::CPAN::Meta" => 0,
    "Test::Perl::Critic" => 0,
    "Test::Pod::Content" => 0,
    "namespace::autoclean" => 0,
    "strict" => 0,
    "utf8" => 0,
    "warnings" => 0
  },
  "TEST_REQUIRES" => {
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.88"
  },
  "VERSION" => "1.007000",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "CPAN::Meta::YAML" => 0,
  "Carp" => 0,
  "Dist::Zilla" => "4.300000",
  "Dist::Zilla::Plugin::ArchiveRelease" => 0,
  "Dist::Zilla::Plugin::Authority" => 0,
  "Dist::Zilla::Plugin::AutoMetaResources" => 0,
  "Dist::Zilla::Plugin::AutoPrereqs" => 0,
  "Dist::Zilla::Plugin::Bootstrap::lib" => 0,
  "Dist::Zilla::Plugin::ConfirmRelease" => 0,
  "Dist::Zilla::Plugin::ConsistentVersionTest" => 0,
  "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
  "Dist::Zilla::Plugin::DualBuilders" => 0,
  "Dist::Zilla::Plugin::EOLTests" => 0,
  "Dist::Zilla::Plugin::ExecDir" => 0,
  "Dist::Zilla::Plugin::ExtraTests" => 0,
  "Dist::Zilla::Plugin::FakeRelease" => 0,
  "Dist::Zilla::Plugin::GatherDir" => 0,
  "Dist::Zilla::Plugin::HasVersionTests" => 0,
  "Dist::Zilla::Plugin::InstallGuide" => 0,
  "Dist::Zilla::Plugin::License" => 0,
  "Dist::Zilla::Plugin::MakeMaker" => 0,
  "Dist::Zilla::Plugin::Manifest" => 0,
  "Dist::Zilla::Plugin::ManifestSkip" => 0,
  "Dist::Zilla::Plugin::MetaConfig" => 0,
  "Dist::Zilla::Plugin::MetaJSON" => 0,
  "Dist::Zilla::Plugin::MetaNoIndex" => 0,
  "Dist::Zilla::Plugin::MetaTests" => 0,
  "Dist::Zilla::Plugin::MetaYAML" => 0,
  "Dist::Zilla::Plugin::MinimumPerl" => 0,
  "Dist::Zilla::Plugin::ModuleBuild" => 0,
  "Dist::Zilla::Plugin::NextRelease" => 0,
  "Dist::Zilla::Plugin::NoTabsTests" => 0,
  "Dist::Zilla::Plugin::PkgVersion" => 0,
  "Dist::Zilla::Plugin::PodCoverageTests" => 0,
  "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
  "Dist::Zilla::Plugin::PodWeaver" => 0,
  "Dist::Zilla::Plugin::PruneCruft" => 0,
  "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
  "Dist::Zilla::Plugin::ReportVersions::Tiny" => 0,
  "Dist::Zilla::Plugin::ShareDir" => 0,
  "Dist::Zilla::Plugin::SpellingCommonMistakesTests" => 0,
  "Dist::Zilla::Plugin::Test::CPAN::Changes" => "0.005",
  "Dist::Zilla::Plugin::Test::Compile" => "1.112400",
  "Dist::Zilla::Plugin::Test::DistManifest" => "v2.0.1",
  "Dist::Zilla::Plugin::Test::MinimumVersion" => "v2.0.1",
  "Dist::Zilla::Plugin::Test::Perl::Critic" => "2.112410",
  "Dist::Zilla::Plugin::Test::Portability" => "v2.0.1",
  "Dist::Zilla::Plugin::Test::Synopsis" => "v2.0.1",
  "Dist::Zilla::Plugin::TestRelease" => 0,
  "Dist::Zilla::Plugin::UploadToCPAN" => 0,
  "Dist::Zilla::Role::MintingProfile::ShareDir" => 0,
  "Dist::Zilla::Role::PluginBundle::Easy" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Module::Build" => "0.3601",
  "Moose" => "2.0",
  "Perl::Critic" => "1.118",
  "Perl::Critic::Bangs" => "1.08",
  "Perl::Critic::Lax" => 0,
  "Perl::Critic::Pulp" => 0,
  "Perl::Critic::StricterSubs" => 0,
  "Perl::Critic::Swift" => 0,
  "Perl::MinimumVersion" => 0,
  "Perl::OSType" => 0,
  "Perl::PrereqScanner" => 0,
  "Pod::Coverage::TrustPod" => 0,
  "Pod::Elemental::Transformer::List" => 0,
  "Pod::Spell" => 0,
  "Pod::Weaver" => "3.101632",
  "Pod::Weaver::Config::Assembler" => 0,
  "Pod::Weaver::Section::Installation" => 0,
  "Pod::Weaver::Section::SeeAlso" => 0,
  "Pod::Weaver::Section::Support" => 0,
  "Pod::Weaver::Section::WarrantyDisclaimer" => 0,
  "Test::CPAN::Meta" => 0,
  "Test::More" => "0.88",
  "Test::Perl::Critic" => 0,
  "Test::Pod::Content" => 0,
  "namespace::autoclean" => 0,
  "strict" => 0,
  "utf8" => 0,
  "warnings" => 0
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);

{
package
MY;
use File::ShareDir::Install qw(postamble);
}


